index.js

```diff
@@ -54,7 +71,19 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let sumOfRatings = 0;
+  let numberOfTitles = 0;
+  if(!movies.length){
+    return sumOfRatings;
+  }
+
+  for (let i = 0; i < movies.length; i++){
+    sumOfRatings += Number(movies[i].imdbRating);
+    numberOfTitles += 1;
```

Comment:
I like this iteration on `numberOfTitles` however if we are confident that our consistent and normalized we could skip this and simply divide `sumOfRatings` by `movies.length`